challenge/public/index.html

```diff
@@ -5,23 +5,32 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta name="theme-color" content="#0000" />
     <meta name="description" content="Web site created using create-react-app" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/styles.css"/>
     <title>HackRU Reactment Sandbox</title>
-    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Comfortaa:bold&display=swap&effect=fire-animation">
     <style>
-        body {
-            font-family: 'Montserrat', sans-serif;
-        }
+      body {
```

Comment:
potentially after you created this pull this project got a main css file under `challenge/public/styles.css`, these styles should be moved in there

Reply:
I've been wanting to pull the updates in but I get the error "Pulling is not possible because you have unmerged files".